### PR TITLE
Generate from range

### PR DESCRIPTION
**Pull request by eruizgar**
## Description
Now you can create number or decimal included in a range of two numbers.

### Testing
- [X] Unit, integration, acceptance tests as appropriate
- [ ] Coverage (>90%)

### Documentation
- [X] Changelog update
- [X] Documentation entry

### Scalastyle
Result of scalastyle execution: 
Processed 26 file(s)
Found 0 errors
Found 0 warnings
Found 0 infos
Finished in 1941 ms

### CODE REVIEW

File: src/test/scala/com/stratio/hermes/utils/HermesTest.scala [hunk 1]
```diff
@@ -139,13 +139,33 @@ class HermesTest extends FlatSpec with Matchers {
...
   }
 
+  it should "generate a random integer when it passed the range " in {
+    val hermesNum = Hermes("")
+    forAll { (n: Int, m: Int) =>
+      hermesNum.Number.numberInRange(n, m) <= m && hermesNum.Number.numberInRange(n, m) >= n
+    }
+    //scalastyle:off
+    numberOfDigitsFromANumber(hermesNum.Number.numberInRange(1,9)) shouldBe 1
+    numberOfDigitsFromANumber(hermesNum.Number.numberInRange(99,10)) shouldBe 2
```
Comment (albertostratio) [inline src/test/scala/com/stratio/hermes/utils/HermesTest.scala]: It shouldn't be numberInRange(10,99)??

**Review by albertostratio — CHANGES_REQUESTED** [review]

File: src/test/scala/com/stratio/hermes/utils/HermesTest.scala (hunk 1 above)
Comment (eruizgar) [reply src/test/scala/com/stratio/hermes/utils/HermesTest.scala]: You can pass the range in both ways, (10,99) and (99,10)

Comment (albertostratio) [reply src/test/scala/com/stratio/hermes/utils/HermesTest.scala]: ok, fair enough

**Review by albertostratio — APPROVED** [review]
LGTM

**Pull request merged**